.example.bot.env

```diff
@@ -0,0 +1 @@
+DISCORD_TOKEN=
```

Comment:
are we sure to go with env file?
maybe toml would be better

Reply:
Working with environment variables is more standardized and simpler. It works well with dockerization (the docker compose puts all the variables from the `.bot.env` in the container environment, no setup is needed on the python side). We will still need some environment variables for later, like for the `postgres` container that is coming later.